docs/Api/HoldingsApi.md

```diff
@@ -0,0 +1,180 @@
+# atrium\HoldingsApi
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**listHoldings**](HoldingsApi.md#listHoldings) | **GET** /users/{user_guid}/holdings | List holdings
+[**listHoldingsByAccount**](HoldingsApi.md#listHoldingsByAccount) | **GET** /users/{user_guid}/accounts/{account_guid}/holdings | List holdings by account
+[**listHoldingsByMember**](HoldingsApi.md#listHoldingsByMember) | **GET** /users/{user_guid}/members/{member_guid}/holdings | List holdings by member
+[**readHolding**](HoldingsApi.md#readHolding) | **GET** /users/{user_guid}/holdings/{holding_guid} | Read holding
+
+
+# **listHoldings**
+> \atrium\model\HoldingsResponseBody listHoldings($user_guid)
+
+List holdings
+
+Use this endpoint to read all holdings associated with a specific user.
```

Comment:
Could you combine these several methods into one? Instead of having `getHoldingsBy<x>` it could be `getHoldings` that would take an associative array of options:

```php
$client->holdings->getHoldings(array("user_guid" => $user_guid, "account_guid" => $account_guid))
```

Reply:
Currently swagger does not provide an easy way to do this in their generators. We could hard-code this exception in, but I'm not sure we want to go that route yet in the 7 language generators we use.

We are currently generating code using OpenAPI 2.0, but 3.0 has been released. With this release, the generators will be updated so it may be easier down the road.